auctions/views.py

```diff
@@ -7,17 +7,25 @@
 from auctions.models import Bid, Claim, Vote
 
 from django.views.generic import TemplateView
-from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
 
 
-class BidStatusView(LoginRequiredMixin, TemplateView):
+class AddonLogin (TemplateView):
+    template_name = 'addon/logon.html'
+
+
+class BidStatusView(UserPassesTestMixin, LoginRequiredMixin, TemplateView):
     """
     Requests for /bid/?url= will receive the HTML form for creating a bid (if
     none exists) or updating the user's existing bid.
 
     url -- url of an OSS issue or bug
     """
     template_name = "addon/widget.html"
+    def test_func(self):
```

Comment:
move `test_func` below the property assignments, and it needs 1 blank line between properties and function declarations/definitions